pyaedt/icepak.py

```diff
@@ -758,14 +758,16 @@ def create_network_block(
             Name of the object to create the block for.
         power : str or var
             Input power.
-        rjc :
+        rjc : float
             RJC value.
-        rjb :
+        rjb : float
             RJB value.
-        gravity_dir :
-            Gravity direction from -X to +Z. Options are ``0`` to ``5``.
-        top :
-            Board bounding value in millimeters of the top face.
+        gravity_dir : int
+            Gravity direction X to Z. Options are ``0`` to ``2``. Determines the orientation of network boundary faces
+        top : float, optional
+            Chosen orientation (X to Z) coordinate value in millimeters of the top face of board.
+            Determines the casing and board side of network.
+            The default is ''0 mm''.
```

Comment:
```suggestion
```
The default should be after the brief description, with more information following.